pyseir/rt/utils.py

```diff
@@ -234,3 +243,27 @@ def align_time_series(series_a, series_b):
         return valid_shifts[np.argmax(xcor)]
     else:
         return 0
+
+
+def load_Rt_result(fips) -> Optional[pd.DataFrame]:
+    """
+    Load the Rt inference result.
+
+    Parameters
+    ----------
+    fips: str
+        State or County FIPS code.
+
+    Returns
+    -------
+    results: pd.DataFrame
+        DataFrame containing the R_t inferences.
+    """
+    if fips in NEW_ORLEANS_FIPS:
```

Comment:
I think it would be helpful to add an easy toggle for this if possible so that we can turn on/off NO fips easily.

possible an environment variable?
maybe
```
DISABLE_NEW_ORLEANS_COMBINED_RT = os.getenv('DISABLE_NEW_ORLEANS_COMBINED_RT', False)
```

then you could do:
```
if not DISABLE_NEW_ORLEANS_COMBINED_RT and fips in NEW_ORLEANS_FIPS:
```

Reply:
We should decide how many of these aggregations we are going to do. It's not clear to me if we should have for every bespoke aggregation we do, or if a single bespoke toggle is enough. I'll punt on this decision for this PR.